lib/rules/function-no-unknown/index.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const fs = require('fs');
+const valueParser = require('postcss-value-parser');
+const functionsListPath = /** @type {string} */ (
+	/** @type {unknown} */ (require('css-functions-list'))
```

Comment:
This package is too old and doesn't contain all functions

Reply:
Oh, do you know some alternative or another way?

Reply:
If the package is insufficient, I think we should contain a function name set internally like the math functions: 🤔 

https://github.com/stylelint/stylelint/blob/b0523d8e76929d8c7224f884b0b256c2657ce977/lib/reference/mathFunctions.js#L3

Any thoughts?

Reply:
This package uses data from MDN.
I cloned it locally and run `npm run generate-list` and got 10 new functions in the diff.
I guess we can make a PR to update the package, unless you want to maintain it inside stylelint. Up to you :)

Reply:
@gnuletik Thanks for the feedback. Could you please open a PR to [niksy/css-functions-list](https://github.com/niksy/css-functions-list), please?
If the repository owner will maintain the package, we can go ahead.

If not, then we will maintain the function list by ourselves. I'm a bit concerned about our maintenance resources... 🤔

Reply:
> @gnuletik How about using `URL` as below?
> 
> ```js
> fs.readFileSync(new URL("file://" + listPath))
> ```

I just made the change. I need approval to run the CI.

Reply:
I think we can have list here, it is not hard to maintenance

Reply:
Done :) https://github.com/niksy/css-functions-list/pull/1

Reply:
New version has been pushed! 😉

Reply:
@niksy Thank you so much! 😊